examples/app-python/README.md

```diff
@@ -0,0 +1,39 @@
+# Example Python Payload Application
+
+This directory contains an example Payload Application using the Python.
+This is not production-ready code, but it is useful for learning how the platform functions.
+
+## Quickstart
+
+In the Antaris Cloud Platform, create a TrueTwin Satellite with a remote payload and download the associated config.
+Place that downloaded zip file in this directory.
+
+Build the app using the following command:
+
+```
+docker build --platform=linux/amd64 -t satos-payload-example-app-python .
+```
+
+Next, we can run the application in a container. The command below assumes that `CONFIG` is set to the name of the downloaded file in your current working directory:
```

Comment:
Meaning of "Downloaded file", we may need to explain.
Downloaded file = path of zip file downloaded for true-twin.

Reply:
Cleaned this up.